lib/livebook/teams.ex

```diff
@@ -249,4 +249,19 @@ defmodule Livebook.Teams do
       []
     end
   end
+
+  def deploy_app(%Team{} = team, %Livebook.Session{} = session) do
+    data = Livebook.Session.get_data(session.pid)
+
+    with {:ok, app_deployment} <- AppDeployment.new(data),
+         {:ok, _} <- Requests.deploy_app(team, app_deployment) do
+      :ok
+    else
+      {:error, %{"errors" => _}} ->
+        {:error, "Something went wrong, try again later or please file a bug if it persists"}
+
+      any ->
+        any
+    end
```

Comment:
Let's use `case`s for clarity where each error comes from. Or perhaps `with {:ok, app_deployment} <- AppDeployment.new(data) do ... end` and then a `case` inside.